jetty-load-generator-client/src/main/java/org/mortbay/jetty/load/generator/LoadGeneratorRunner.java

```diff
@@ -139,25 +144,28 @@ private void handleResource( Resource resource )
             // it's a group so we can request in parallel but wait all responses before next step
             ExecutorService executorService = Executors.newWorkStealingPool();
```

Comment:
Not sure why we want to create a new `ExecutorService` here ? Can't we just use the existing one ?